### DISCUSSION TITLE
ram dimm sensor

### DISCUSSION THREAD

**Discussion by reddev94** [Rem0o/FanControl.Releases#1276] [Ideas]
hi, there is a way to use ram temperature sensor as source when creating a custom curve ?

**Comment by reddev94** [comment]
any update ?

**Comment by Fr0stX76** [comment]
If your RAM is supported by HWinfo, you could use the plugin

https://github.com/Rem0o/FanControl.HWInfo

**Reply by matheo742** [reply]
I've installed the plugin ; on Hwinfo sensors are working,  I have 1 per stick and on Fancontrol not

**Reply by Fr0stX76** [reply]
Did you add both sensors individually in the gadget section of hwinfo?

Also make sure to rename one in Hwinfo if they have the same name.

Edit. Did you follow those instructions?

1-Copy the FanControl.HWInfo.dll into FanControl's "Plugins" folder. You might need to unblock it in its properties.

2-Go the HWInfo's "Configure Sensors" option menu, then to the "HWInfo Gadget" tab. Have the "Enable reporting to gadget" checkbox checked and select "Report value in Gadget" for all sensors you want to import.

**Reply by matheo742** [reply]
oKAY
Do I need to have hwinfo open to use these sensor? 
I have this bug ,there an application error (pop-up) and sensors show 0 while hwinfo is open
[log.txt](https://github.com/user-attachments/files/17009558/log.txt)

**Reply by Fr0stX76** [reply]
Yes, in sensor mode at the very minimum

**Reply by matheo742** [reply]
i got error application described in log.txt

**Reply by Fr0stX76** [reply]
From that log file.

1- Did you put the plugin dll in a subfolder of fan control? The folder needs to be named plugins

> Drop your dll into the Plugins folder of FanControl. At startup, it will find IPlugins implementations by reflection.

2- Have you made sure the .dll is not blocked?

![2020-02-27_14h40_57(4)(1)-448462923](https://github.com/user-attachments/assets/9a93e992-274b-42c9-acb0-c66f745b040b)

**Reply by matheo742** [reply]
Folder : [image](https://github.com/user-attachments/assets/9cf822ed-91f9-406e-b181-3e8b405fdd1b)
![image](https://github.com/user-attachments/assets/1ef378fe-1bd1-4e97-a2a8-0c26b1011c45)

DLL not locked
the folder hwinfo is ready only

**Reply by Fr0stX76** [reply]
The dll need to be in the plugins folder directy, not in an hwinfo sub-folder

**Reply by Rem0o** [reply]
@Fr0stX76  subfolder does work.

From the log it seems like it's a formatting problem.   @matheo742 has comma separator for decimals, which doesn't parse.

**Reply by matheo742** [reply]
because i'm french and using "," instead of "." ??

**Reply by Fr0stX76** [reply]
@Rem0o, good to know.

@matheo742 oui exactement. Juste a changer ceci pour un "." dans HWinfo

![image](https://github.com/user-attachments/assets/b4e9e106-a0fe-40f0-b5fd-c331da7f8e65)

**Reply by matheo742** [reply]
Perfect Values are good !
thanks :)
